### DISCUSSION TITLE
2022/07/04/build-cors-anywhere/

### DISCUSSION THREAD

**Discussion by giscus** [argvchs/giscus-comments#11] [Announcements]
# 2022/07/04/build-cors-anywhere/

https://argvchs.github.io/2022/07/04/build-cors-anywhere/

**Comment by ilovend** [comment]
啊咧,heroku好像不免费了,怎么办呢?

**Reply by argvchs** [reply]
所以现在把评论改成 Giscus 了啊（

**Reply by ilovend** [reply]
嗷嗷
